### PR TITLE
[dashboard] Add a few minor missing pages

### PR DESCRIPTION
**Pull request by jankeromnes**
Missing from https://github.com/gitpod-com/gitpod/issues/5233:

- [x] 404 page (sadly has no actual 404 HTTP status, which is required for SEO)
- [x] Sorry page (fixes https://github.com/gitpod-io/gitpod/issues/3757)
- [x] Blocked page (fixes https://github.com/gitpod-io/gitpod/issues/3588)

### CODE REVIEW

**Comment by jankeromnes** [comment]
How to test:

- 404 page: https://jx-minor-missing-pages.staging.gitpod-dev.com/asdfasdf
- Sorry page: https://jx-minor-missing-pages.staging.gitpod-dev.com/sorry#Error%20502
- Blocked page: https://jx-minor-missing-pages.staging.gitpod-dev.com/blocked

**Comment by jankeromnes** [comment]
Current WIP state:

Screenshot 2021-04-06 at 14 37 30

Screenshot 2021-04-06 at 14 37 17

Screenshot 2021-04-06 at 14 37 40

**Comment by jankeromnes** [comment]
Current state:

Screenshot 2021-04-06 at 15 34 20

Screenshot 2021-04-06 at 15 34 30

Screenshot 2021-04-06 at 15 34 11

File: components/dashboard/src/App.tsx [hunk 1]
```diff
@@ -106,6 +115,18 @@ function App() {
                 <Route path={["/admin"]} exact>
                     <Redirect to="/admin/users"/>
                 </Route>
+                <Route path="/sorry" exact>
+                    <div className="mt-48 text-center">
+                        <h1 className="text-gray-500 text-3xl">Oh, no! Something went wrong!</h1>
+                        <p className="mt-4 text-lg text-gitpod-red">{decodeURIComponent(getURLHash())}</p>
+                    </div>
+                </Route>
+                <Route path="*" /* status={404} */>
```
Comment (jankeromnes) [inline components/dashboard/src/App.tsx:124]: Would be nice to send an actual HTTP 404 status code here, but this requires some server-side hook.

**Comment by csweichel** [comment]
image

some misalignment on the "blocked" page

**Comment by jankeromnes** [comment]
> some misalignment on the "blocked" page

Thanks. Fixed.

**Review by svenefftinge — APPROVED** [review]
LGTM

**Review by gtsiolis — COMMENTED** [review]
Thanks @jankeromnes for [adding instructions how to check these changes](https://github.com/gitpod-io/gitpod/pull/3760#issuecomment-814087857)!

Looks like a great MVC! 🧱

**Review by gtsiolis — APPROVED** [review]

**Pull request merged**